src/newItinerary/Itinerary.story.mdx

```diff
@@ -0,0 +1,219 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { BoostIcon } from '../icon/boostIcon'
+import { Distances, Proximity, ProximityDisplay } from '../proximity'
+import { Stepper, StepperDisplay } from '../stepper'
+import { Addon, Connection, HiddenStop, HiddenStops, Itinerary, Lines, Place } from './index'
+
+<Meta title="Widgets/NewItinerary" />
+
+# **Itinerary**
+
+### Search result
+
+<Canvas>
+  <Story name="Search">
+    <Itinerary>
+      <Place
+        label="Villepinte"
+        time="12:50"
+        proximity={<Proximity value={Distances.FAR} title="You'll have to take transports" />}
+      />
+      <Place
+        label="Lyon"
+        time="18:00"
+        proximity={<Proximity value={Distances.CLOSE} title="You'll get there by foot" />}
+      />
+    </Itinerary>
+  </Story>
+</Canvas>
+
+### Ride details - carpooling
+
+<Canvas>
+  <Story name="Ride details - Carpool">
+    <Itinerary>
+      <Place
+        label="Villepinte"
+        time="12:50"
+        proximity={<Proximity value={Distances.FAR} title="3.7 km from your departure" display={ProximityDisplay.LABEL} />}
+        href="https://blablacar.fr"
+      />
+      <Place
+        label="Lyon"
+        time="18:00"
+        proximity={<Proximity value={Distances.CLOSE} title="400m from your arrival" display={ProximityDisplay.LABEL} />}
+        href="https://blablacar.fr"
+      />
+      <Addon label="Marseille" />
+    </Itinerary>
+  </Story>
+</Canvas>
+
+### Ride details - bus
+
+<Canvas>
+  <Story name="Ride details - Bus">
+    <Itinerary aria-label="Your ride from Paris to Marseille">
+      <Addon label="Lille" aria-label="Bus comes from Lille" />
+      <Place
+        aria-label="Pickup at 09:00 at Gare de Lyon, Paris"
+        time="09:00"
+        label="Paris"
+        subLabel="Gare de lyon"
+      />
+      <HiddenStops label="2 stopovers" aria-label="Click to show intermediate stops">
+        <HiddenStop label="Auxerre" aria-label="Bus stops in Auxerre" />
+        <HiddenStop label="Dijon" aria-label="Bus stops in Dijon" />
+      </HiddenStops>
+      <Place time="12:00" label="Lyon" subLabel="Gare Perrache" aria-label="Drop off the vehicle at 12:00 at Gare Perrache, Lyon" />
```

Comment:
`aria-label` on `Place` should only content complementary text (Arrival/Pickup/Dropoff/arrival) as other children in the `<li>` will be readed. Otherwise we would had to hide children to avoid redundant infos (`aria-hidden`).